clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -124,6 +124,13 @@ public class ConsumerConfig extends AbstractConfig {
         "the group coordinator will pick one. This configuration is applied only if <code>group.protocol</code> is " +
         "set to \"consumer\".";
 
+    /**
+     * <code>group.local.assignors</code>
+     */
+    public static final String GROUP_LOCAL_ASSIGNORS_CONFIG = "group.local.assignors";
```

Comment:
we don't have any use case of this config. Is it necessary to add configs check now?

@philipnee WDYT?